elements/rh-secondary-nav/rh-secondary-nav-lightdom.css

```
@@ -1,26 +1,27 @@
 rh-secondary-nav {
+  --_rh-secondary-nav-max-height: max-content;
```

'private' vars don't need prefixes, since unlike 'public' vars, they are assigned on the host (that distinction and the 'why' should go in the wiki)
```suggestion
  --_max-height: max-content;
```
etc

---

Fixed in eb2256e4152e17de05b8cbd4bb5997cf55fbcbdd